### PR TITLE
Add another null guard for member

### PR DESCRIPTION
**Pull request by dbkr**
To hopefully fix https://github.com/vector-im/element-web/issues/21319

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Add another null guard for member ([\#7984](https://github.com/matrix-org/matrix-react-sdk/pull/7984)). Fixes vector-im/element-web#21319.

Preview: https://pr7984--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: src/components/views/messages/SenderProfile.tsx [hunk 1]
```diff
@@ -113,7 +113,7 @@ export default class SenderProfile extends React.Component<IProps, IState> {
         let member = mxEvent.sender;
         if (SettingsStore.getValue("feature_use_only_current_profiles")) {
             const room = MatrixClientPeg.get().getRoom(mxEvent.getRoomId());
-            if (room) {
+            if (room && member) {
                 member = room.getMember(member.userId);
```
Comment (t3chguy) [inline src/components/views/messages/SenderProfile.tsx:117]:
```suggestion
            if (room) {
                member = room.getMember(mxEvent.getSender());
```

Comment (dbkr) [reply src/components/views/messages/SenderProfile.tsx:117]: Wouldn't this pass null to getMember though which seems a bit weird?

Comment (t3chguy) [reply src/components/views/messages/SenderProfile.tsx:117]: getSender returns the MXID even if there is no sender/member object

Comment (dbkr) [reply src/components/views/messages/SenderProfile.tsx:117]: oh, TIL

**Review by gsouquet — APPROVED** [review]

**Pull request merged**